releases/release-9.0.0.md

```diff
@@ -0,0 +1,424 @@
+    <td>[The microservice mode](/pd-microservices.md) supported by PD is generally available (GA) (introduced in v8.0.0)</td>
+    <td>The PD microservice mode decouples different functional modules of PD into independent services, improving system scalability, stability, and deployment flexibility. It provides a more robust architectural foundation for large-scale cluster deployments.</td>
+  </tr>
+    <tr>
+    <td>Point-In-Time-Recovery (PITR) Now Supports Recovery from Compacted Log Backups for Faster Restores</td>
+    <td>Starting from v9.0.0, the log backup feature provides offline compaction capabilities, converting unstructured log backup data into structured SST files. These SST files can now recovered into the cluster much more quickly than reapplying the original logs, delivering improved recovery performance.</td>
+  </tr>
+  <tr>
+    <td rowspan="1">Reliability and availability</td>
+    <td>TiProxy traffic replay feature is generally available (GA) (introduced in v8.4.0)</td>
+    <td>Before performing critical operations such as cluster upgrades, migrations, or deployment changes, use TiProxy to capture the real workload from the production TiDB cluster and replay it on the target test cluster. It helps validate performance and ensure the success of the changes.</td>
+  </tr>
+  <tr>
+    <td rowspan="3">DB Operations and Observability</td>
+    <td> TiDB Workload Repository </td>
+    <td>将数据库运行时的历史状态持久化，能够显著提升历史故障和性能问题的诊断效率，帮助用户快速定位并优化问题，同时为健康检查和自动调优提供关键的数据基础。</td>
+  </tr>
+  <tr>
+    <td>TiDB Index Advisor tw@Oreoxmt</td>
+    <td>TiDB Index Advisor analyzes actual query workloads to automatically identify missing or redundant indexes. It helps you optimize indexes without requiring deep knowledge of your application. This feature reduces the cost of manual analysis and tuning, and improves query performance and system stability.</td>
+  </tr>
+  <tr>
+    <td>SQL cross-AZ traffic monitoring tw@Oreoxmt</td>
+    <td>This feature helps you identify cross-availability zone (AZ) network traffic caused by SQL queries in a TiDB cluster. It enables you to analyze traffic sources, optimize deployment architecture, and control cross-AZ data transfer costs in cloud environments, thus improving resource efficiency and cost visibility.</td>
+  </tr>
+  <tr>
+    <td rowspan="3">Data Migration</td>
+    <td>Query argument redaction in Data Migration (DM) logs tw@Oreoxmt</td>
+    <td>Data Migration (DM) introduces the <code>redact-info-log</code> configuration item to redact query arguments in DM logs, preventing sensitive data from appearing in logs.</td>
+  </tr>
+  <tr>
+    <td>TiDB Lightning supports compatibility with <code>sql_require_primary_key=ON</code> in TiDB tw@Oreoxmt</td>
+    <td>When the <code>sql_require_primary_key=ON</code> system variable is enabled in TiDB, the internal error log table now includes a primary key to prevent table creation failures during import.</td>
```

Comment:
```suggestion
    <td>When the <code>sql_require_primary_key</code> system variable is enabled in TiDB, TiDB Lightning automatically adds a default primary key to its internal error-logging and conflict-detection tables during data import to prevent table creation failures.</td>
```